adam/curriculum/preposition_curriculum.py

```diff
@@ -809,6 +877,53 @@ def _make_in_front_tests(
     )
 
 
+def _make_near_tests(
+    num_samples: int = 5, *, noise_objects: bool = True
+) -> Phase1InstanceGroup:
+    figure_0 = standard_object("figure_0", THING, banned_properties=[HOLLOW])
+    figure_1 = standard_object("figure_1", THING, banned_properties=[HOLLOW])
+    ground_0 = standard_object("ground_0", THING, banned_properties=[HOLLOW])
+    ground_1 = standard_object("ground_1", THING, banned_properties=[HOLLOW])
+
+    speaker = standard_object("speaker", MOM, added_properties=[IS_SPEAKER])
```

Comment:
Same thing with the speaker and addressee here, they aren't needed to establish the near relationship.